Sources/FirebaseFirestore/DocumentSnapshot+Swift.swift

```diff
@@ -9,9 +9,37 @@ import CxxShim
 import Foundation
 
 public typealias DocumentSnapshot = firebase.firestore.DocumentSnapshot
+public typealias ServerTimestampBehavior = firebase.firestore.DocumentSnapshot.ServerTimestampBehavior
+public typealias MapFieldValue = firebase.firestore.MapFieldValue
+
+public typealias GeoPoint = firebase.firestore.GeoPoint
+public typealias FieldValue = firebase.firestore.FieldValue
```

Comment:
These all had to become public types due to the surface area of the new set of codable APIs we've added.